src/assets/scripts/client/InputController.js

```diff
@@ -289,14 +309,14 @@ export default class InputController {
         // TODO: this should use early returns instead of the else if
```

Comment:
can you update this `TODO` to say: _This should use a `switch` on `event.which` instead of `if/else if`_

Reply:
done